test/src/686-azureblob_s3/main

```diff
@@ -0,0 +1,92 @@
+#
+# Test for running CVMFS against an Azure S3 implementation
+#
+# In this test we create our own Azurite blob service.
+# require az-cli and azurite to be installed, see install_azurite()
+# keys are hard-coded dev keys in azurite, so safe to have in code
+
+cvmfs_test_name="Azure blob on azurite"
+cvmfs_test_autofs_on_startup=false
+cvmfs_test_suites="quick"
+
+cleanup() {
+  echo "*** cleaning up"
+  sudo cvmfs_server rmfs -f azurite.cvmfs.io
+  if [ -f azurite_pid ]; then
+    sudo kill -9 $(cat azurite_pid)
+  fi
+}
+
+create_azurite_config() {
+  tee cvmfs_azurite.conf > /dev/null << EOF
+CVMFS_S3_HOST=127.0.0.1:10000
+CVMFS_S3_ACCESS_KEY=devstoreaccount1
+CVMFS_S3_SECRET_KEY=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==
+CVMFS_S3_BUCKET=devstoreaccount1/test
+CVMFS_S3_DNS_BUCKETS=false
+CVMFS_S3_FLAVOR=azure
+EOF
+}
+
+install_azurite() {
```

Comment:
Let's put that into the centos7_x86_64_setup.sh file.  If this can be installed on Ubuntu, too, it would be great to add it there as well since we are testing S3 support on EL7 and Ubuntu.

Reply:
now put into EL7 setup; will test Ubuntu later on...